### ISSUE TITLE
Constraining range in multi label classification problem

### ISSUE THREAD

**Issue by Demirrr** [cvxgrp/cvxpylayers#116]
Dear all,

I would like to use cvxplayers to constrain the range of outputs in multi-label classification problem sample-wise. However, I seem to fail to do so. Next I would like to define the problem setting.

Let x \in R^n, y \in {0,1}^k,c in {0,1}^k, and W \in R^{k x n} denote an input, a binary label, binary constraint vector, and a transformation matrix. Predictions are computed by normalizing the log probabilites via sigmoid function, i.e,. y_hat = sigmoid(Wx). Assume that we find W by minimizing the cross entropy loss function.Hence, we deal with standard multi-label classification problem.

1. If the i-th value in c equals 0 (c_i =0), then we can ensure that y_i=0.
2. If c_i=1, then we can not claim whether y_i=1 or y_i=0.

I was wondering whether such sample-wise constraints can be implemented via cvxpylayers.

Cheers
